python/tvm/relay/frontend/tensorflow.py

```diff
@@ -788,52 +797,21 @@ def _impl(inputs, attr, params, mod):
 
 def _tensor_array():
     def _impl(inputs, attr, params, prelude):
-        try:
-            from tensorflow.python.framework import tensor_util
-        except ImportError as e:
-            raise ImportError(
-                "Unable to import tensorflow which is required {}".format(e))
-
         dtype_str = attr.get('dtype').name
         assert not attr["dynamic_size"], "Dynamic size tensor array is " \
                                          "not supported in TVM yet."
 
-        raw_elem_shape = tensor_util.TensorShapeProtoToList(attr['element_shape'])
-        elem_shape = []
-        for dim in raw_elem_shape:
-            if dim < 0:
-                elem_shape.append(Any())
-            else:
-                elem_shape.append(dim)
-
-        if elem_shape:
-            # Element shape is specified.
-            # Directly create static tensor array with given shape.
+        if attr['identical_element_shapes']:
+            assert "shape" in attr, "Shape is required for static tensor array."
```

Comment:
Should it be element_shape?
https://github.com/tensorflow/tensorflow/blob/a34091e538540aad57a7a941575538944f38db24/tensorflow/core/kernels/tensor_array_ops.cc#L165

Reply:
shape is set manually here.